core-customize/build.gradle.kts

```diff
@@ -104,6 +92,57 @@ tasks.register<Exec>("symlinkSolrConfigForLocalDevelopment") {
     workingDir("hybris/config/solr/instances/cloud")
 }
 
+// ant modulegen -Dinput.module=accelerator -Dinput.name=demoshop -Dinput.package=com.demo.shop
+tasks.register<HybrisAntTask>("generateDemoStorefront") {
+    dependsOn("bootstrapPlatform", "createDefaultConfig")
+
+    args("modulegen")
+    antProperty("input.module", "accelerator")
+    antProperty("input.name", "demoshop")
+    antProperty("input.package", "com.demo.shop")
+}
+
+// ant extgen -Dinput.template=yacceleratorordermanagement -Dinput.name=demoshopordermanagement -Dinput.package=com.demo.shop.ordermanagement
+tasks.register<HybrisAntTask>("generateDemoOrderManagment") {
+    dependsOn("bootstrapPlatform", "createDefaultConfig")
+
+    args("extgen")
+    antProperty("input.template", "yacceleratorordermanagement")
+    antProperty("input.name", "demoshopordermanagement")
+    antProperty("input.package", "com.demo.shop.ordermanagement")
+}
+
+// ant extgen -Dinput.template=yocc -Dinput.name=demoshopocc -Dinput.package=com.demo.shop.occ
+tasks.register<HybrisAntTask>("generateDemoOcc") {
```

Comment:
Why do you generate two occ extensions? Just to show how it works?

Reply:
As discussed Custom OCC extensions are not strictly needed if you use the CX recipes for demo purposes... But in a Spartacus project that’s most likely something that will be required C.f. https://help.sap.com/viewer/9d346683b0084da2938be8a285c0c27a/2005/en-US/d46d19516961438f8939718e87ed787b.html

There’s also the the demoshopordermanagement extension which replaces the demoshopfulfilmentprocess extension in the `cx` recipe